cmd/tkn/main.go

```diff
@@ -26,7 +29,25 @@ func main() {
 	tp := &cli.TektonParams{}
 	tkn := cmd.Root(tp)
 
-	if err := tkn.Execute(); err != nil {
+	args := os.Args[1:]
+	cmd, _, _ := tkn.Find(args)
+	if cmd != nil && cmd == tkn {
+		pluginCmd := "tkn-" + os.Args[1]
```

Comment:
If I do only `tkn` with no args then this will break here. index out of range.
Should we handle this? If there is no argument then skip this.

Reply:
fixed it by checking `len(args)` :wink: